api/envoy/service/rate_limit_quota/v3/rlqs.proto

```diff
@@ -78,6 +80,9 @@ message RateLimitQuotaUsageReports {
 
     // Requests throttled.
     uint64 num_requests_denied = 4;
+
+    // An optional info for the list of Envoy instances that send the usage report ``BucketQuotaUsage``.
```

Comment:
I'm not sure if you want to send this on every request, this message can get substantially large CC @yanavlasov

Reply:
Yes, You are right. This is designed/recommended to be only on first report (similar to the domain name [here](https://github.com/envoyproxy/envoy/blob/879ff9cb7e6cd522138f11e3b10f96dd5ae8f67d/api/envoy/service/rate_limit_quota/v3/rlqs.proto#L95) )

I was on the fence of adding this recommendation to the code comment as I was thinking this eventually is really up to the customer who set this field.  But on second thought, let me add this to the comment to make it clear.